README.md

```diff
@@ -44,6 +43,9 @@ cd SmartSim-Scaling
 pip install -r requirements.txt
 ```
 
+You will need to install ``mpi4py`` in your python environment. The install instructions
```

Comment:
could we add mpi4py to our `requirements.txt`?

Reply:
if I add mpi4py to requirements.txt, will the user still need to download it via the mpi4py docs? I am thinking yes bc they might need to install it on their machine and the requirements.txt is for python env?